Traceback (most recent call last):
  File "/home/frederikwr/anaconda3/envs/WindBro/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/frederikwr/anaconda3/envs/WindBro/lib/python3.11/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frederikwr/anaconda3/envs/WindBro/lib/python3.11/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frederikwr/anaconda3/envs/WindBro/lib/python3.11/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/frederikwr/anaconda3/envs/WindBro/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/frederikwr/anaconda3/envs/WindBro/lib/python3.11/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/frederikwr/anaconda3/envs/WindBro/lib/python3.11/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/frederikwr/anaconda3/envs/WindBro/lib/python3.11/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
V, F = load_and_normalize_obj("../assets/eiffel.obj", total_height = 1)
print("V: ",V,"\n\nF: ", F)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mFileNotFoundError[0m                         Traceback (most recent call last)
Cell [0;32mIn[3], line 1[0m
[0;32m----> 1[0m V, F [38;5;241m=[39m load_and_normalize_obj([38;5;124m"[39m[38;5;124m../assets/eiffel.obj[39m[38;5;124m"[39m, total_height [38;5;241m=[39m [38;5;241m1[39m)
[1;32m      2[0m [38;5;28mprint[39m([38;5;124m"[39m[38;5;124mV: [39m[38;5;124m"[39m,V,[38;5;124m"[39m[38;5;130;01m\n[39;00m[38;5;130;01m\n[39;00m[38;5;124mF: [39m[38;5;124m"[39m, F)

File [0;32m~/Dropbox/DTU Special Course/src/utils.py:335[0m, in [0;36mload_and_normalize_obj[0;34m(file_path, total_height)[0m
[1;32m    333[0m [38;5;28;01mdef[39;00m [38;5;21mload_and_normalize_obj[39m(file_path, total_height [38;5;241m=[39m [38;5;241m1[39m):
[1;32m    334[0m     V, F [38;5;241m=[39m [], []
[0;32m--> 335[0m     [38;5;28;01mwith[39;00m [38;5;28mopen[39m(file_path) [38;5;28;01mas[39;00m f:
[1;32m    336[0m         [38;5;28;01mfor[39;00m line [38;5;129;01min[39;00m f[38;5;241m.[39mreadlines():
[1;32m    337[0m             [38;5;28;01mif[39;00m line[38;5;241m.[39mstartswith([38;5;124m'[39m[38;5;124m#[39m[38;5;124m'[39m):

[0;31mFileNotFoundError[0m: [Errno 2] No such file or directory: '../assets/eiffel.obj'
FileNotFoundError: [Errno 2] No such file or directory: '../assets/eiffel.obj'

